styles/style.css

```diff
@@ -0,0 +1,855 @@
+.cover-img-container {
+  position: relative;
+}
+
+.heroe-img-category h3 {
+  position: absolute;
+  bottom: 3px;
+  background-color: #c2d4d7;
+  padding: 3px 6px 3px 6px;
+  color: #333;
+  font-size: 12px;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+.cover-img-category h3 {
+  position: absolute;
+  bottom: 3px;
+  background-color: rgba(255, 255, 255, 0.9);
+  padding: 8px 10px 8px 10px;
+  color: #f72210;
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+.heroe-img a,
+.heroe-img img {
+  width: 100%;
+  height: 100%;
+}
+
+.heroe-content {
+  margin: 15px 0 23px;
+}
+
+.heroe-content a {
+  color: rgba(3, 3, 3, 0.5);
+  font-size: 15px;
+  font-weight: 700;
+}
+
+.sign-up {
+  max-width: 300px;
+  margin: 0 auto 30px;
+  border: 1px solid #e5e5e5;
+  background-color: #f4f5f6;
+  padding: 20px;
+}
+
+.sign-up-title {
+  font-size: 19px;
+  font-weight: 700;
+  align-items: center;
+  margin-bottom: 15px;
+}
+
+.sign-up-title img,
+.in-the-magazine-title img {
+  background-color: #f72310;
+  height: 28px;
+  width: 29px;
+  margin-right: 10px;
+}
+
+.in-the-magazine-title {
+  align-items: center;
+  justify-content: center;
+  color: #333;
+  font-size: 19px;
+  font-weight: 700;
+  margin-bottom: 15px;
+  padding-bottom: 10px;
+  border-bottom: 1px solid rgba(3, 3, 3, 0.12);
+}
+
+.weather a > img {
+  height: 24px;
+  vertical-align: middle;
+}
+
+.sign-up-form input {
+  width: 100%;
+  height: 32px;
+  line-height: 32px;
+}
+
+.sign-up-form :nth-child(2) {
+  margin-top: 8px;
+  background-color: #f61602;
+  color: white;
+}
+
+.see-features {
+  color: black;
+  text-align: center;
+  padding: 10px;
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.mag-cat {
+  background-color: #f72210;
+  padding: 5px 9px;
+  color: white;
+  font-size: 14px;
+  font-weight: 700;
+}
+
+.article-category a {
+  padding: 5px 9px;
+  color: #f72210;
+  font-size: 14px;
+  font-weight: 700;
+}
+
+.article-category {
+  margin: 10px 0 20px;
+}
+
+h4 a:hover {
+  color: black;
+  text-decoration: underline #f72210;
+}
+
+.article-category :nth-child(2) {
+  display: inline-block;
+  margin-top: 5px;
+}
+
+.pick-side-info a:hover {
+  text-decoration: none;
+  color: #f92859;
+}
+
+.article-category :nth-child(2) a {
+  padding: 0;
+}
+
+.mag-article-content h3 {
+  font-size: 14px;
+  margin-bottom: 30px;
+  font-weight: 700;
+  color: black;
+}
+
+.last-art-img-category h3 {
+  position: absolute;
+  bottom: 3px;
+  color: rgba(255, 255, 255, 0.9);
+  padding: 8px 10px 8px 10px;
+  background-color: #f72210;
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+.last-mag-discription {
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+  margin: 10px 0 30px;
+}
+
+.subscribe-header h2 {
+  color: #f72210;
+  font-size: 16px;
+  font-weight: 700;
+  padding-bottom: 25px;
+  text-transform: uppercase;
+}
+
+.subscribe-section {
+  flex-direction: column;
+}
```

Comment:
This could be achieved with bootstrap. ([Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options))